Sources/Optimizely+Decide/OptimizelyDecideOption.swift

```diff
@@ -0,0 +1,36 @@
+/****************************************************************************
+* Copyright 2020, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+    
+
+import Foundation
+
+/// Options controlling flag decisions.
+public enum OptimizelyDecideOption {
```

Comment:
I think this should conform to `RawOptionSet` protocol? What do you say?

Reply:
A good point. I'll take a look if it's a better choice.

Reply:
Will keep it as an array. All of the extended feature of OptionSet is not used here. Array of a few options will keep it simple.